Common/src/CConfig.cpp

```diff
@@ -2903,6 +2900,10 @@ void CConfig::SetConfig_Parsing(istream& config_buffer){
                              "and it also applies to discrete adjoint problems.\n\n");
           if (!option_name.compare("WRT_MESH_QUALITY"))
             newString.append("WRT_MESH_QUALITY is deprecated. Use VOLUME_OUTPUT= (MESH_QUALITY, ...) instead.\n\n");
+          if (!option_name.compare("VISUALIZE_SURFACE_DEF"))
```

Comment:
Thanks for going through and removing deprecated code!

Reply:
:+1: I will go through the tutorial cases too to remove those options from there.

Reply:
@TobiKattmann did it for me xD